protocol/chainlib/tendermintRPC.go

```diff
@@ -61,7 +61,12 @@ func (apip *TendermintChainParser) getSupportedApi(name, connectionType string)
 
 func (apip *TendermintChainParser) CraftMessage(parsing *spectypes.ParseDirective, connectionType string, craftData *CraftData, metadata []pairingtypes.Metadata) (ChainMessageForSend, error) {
 	if craftData != nil {
-		chainMessage, err := apip.ParseMsg("", craftData.Data, craftData.ConnectionType, metadata, extensionslib.ExtensionInfo{LatestBlock: 0})
+		path := craftData.Path
+		if craftData.InternalPath != "" {
+			path = craftData.InternalPath
```

Comment:
same Q, do we overwrite with internal path or append it to the path

Reply:
Overwrite, read previous comment